std_msgs/Header header
int32 x_scale
int32 y_scale
roboenvcv/RegionOfInterestInfo[] infos